### PR TITLE
[9.0] web_action_conditionable: migrate to 9.0 (WIP)

### PR DESCRIPTION
**Pull request by ovnicraft**
* [ ] check framework about `is_action_enabled()`

### CODE REVIEW

File: web_action_conditionable/__openerp__.py [hunk 1]
```diff
@@ -1,13 +1,16 @@
...
+# © 2016 Cristian Salamea
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
 
 {
-    "name": 'web_action_conditionable',
-    "version": "8.0.0.1.0",
-    "depends": [
-        'base',
+    'name': 'Conditionable states',
+    'version': '9.0.0.1.0',
```
Comment (zakiuu) [inline web_action_conditionable/__openerp__.py:7]: @ovnicraft 'version': '9.0.1.0.0'

**Review by zakiuu — CHANGES_REQUESTED** [review]

**Comment by zakiuu** [comment]
@ovnicraft Are you willing to finish the PR?

**Comment by ovnicraft** [comment]
I consider more important migration to 10 and 11.

Closing this.

**Comment by zakiuu** [comment]
@ovnicraft thanks for the reply. You can review this one if you have time #861.
@pedrobaeza

**Pull request closed without merging**